bin/build_cmd.mli

```diff
@@ -1,5 +1,10 @@
 open Import
 
+val build_via_rpc_server
+  :  print_on_success:bool
+  -> Dune_lang.Dep_conf.t list
```

Comment:
It's been a long time since I hacked in Dune, but that type seems more generic than "targets", maybe a labelled argument would make it clearer. 

```suggestion
  -> targets:Dune_lang.Dep_conf.t list
```